package.json

```diff
@@ -4,5 +4,17 @@
   "main": "index.html",
   "repository": "https://github.com/MustafaTRKYLMZ/my_favorite_movies_web_app_with_vanilla.js.git",
   "author": "MustafaTRKYLMZ <cana66331@gmail.com>",
-  "license": "MIT"
+  "license": "MIT",
+  "scripts": {
+    "start": "open index.html",
+    "lint": "run lint"
+  },
+  "devDependencies": {
+    "eslint": "^7.32.0 || ^8.2.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
+    "eslint-plugin-promise": "^6.0.0"
```

Comment:
If you opt to not use eslint with the airbnb plugin it is best to take it out here, to avoid questions about it.

Note that `eslint-config-standard` is for a different Style Guide than AirBnb. So not needed here.

The other plugins will also have no effect until you enable them in `eslintrc.js`. In view of time I think it would be wise to stick to the base eslint for now and remove all unused plugins from package.json.